velox/substrait/tests/Substrait2VeloxPlanConversionTest.cpp

```diff
@@ -29,260 +30,454 @@ using namespace facebook::velox;
 using namespace facebook::velox::test;
 using namespace facebook::velox::connector::hive;
 using namespace facebook::velox::exec;
+namespace vestrait = facebook::velox::substrait;
 
 class Substrait2VeloxPlanConversionTest
     : public exec::test::HiveConnectorTestBase {
  protected:
-  std::vector<std::shared_ptr<facebook::velox::connector::ConnectorSplit>>
-  makeSplits(
-      const facebook::velox::substrait::SubstraitVeloxPlanConverter& converter,
-      std::shared_ptr<const core::PlanNode> planNode) {
-    const auto& splitInfos = converter.splitInfos();
-    auto leafPlanNodeIds = planNode->leafPlanNodeIds();
-    // Only one leaf node is expected here.
-    EXPECT_EQ(1, leafPlanNodeIds.size());
-    const auto& splitInfo = splitInfos.at(*leafPlanNodeIds.begin());
-
-    const auto& paths = splitInfo->paths;
-    const auto& starts = splitInfo->starts;
-    const auto& lengths = splitInfo->lengths;
-    const auto fileFormat = splitInfo->format;
-
-    std::vector<std::shared_ptr<facebook::velox::connector::ConnectorSplit>>
-        splits;
-    splits.reserve(paths.size());
-
-    for (int i = 0; i < paths.size(); i++) {
-      auto path = fmt::format("{}{}", tmpDir_->path, paths[i]);
-      auto start = starts[i];
-      auto length = lengths[i];
-      auto split = facebook::velox::exec::test::HiveConnectorSplitBuilder(path)
-                       .fileFormat(fileFormat)
-                       .start(start)
-                       .length(length)
-                       .build();
-      splits.emplace_back(split);
+  class VeloxConverter {
+   public:
+    // This class is an iterator for Velox computing.
```

Comment:
Why do we need this? Can we use the original code?

Reply:
> Why do we need this? Can we use the original code?
Another PR about UT will solve this comments.